src/creatures/players/player.cpp

```diff
@@ -4612,9 +4612,62 @@ void Player::setPremiumDays(int32_t v)
 	sendBasicData();
 }
 
-void Player::setTibiaCoins(int32_t v)
+void Player::setStoreCoins(int32_t coins, CoinType_t coinType)
```

Comment:
Try to break in setCoins / setTournamentCoin

Reply:
Great suggestion, I'll do it.